test/ccall.jl

```diff
@@ -1535,3 +1535,11 @@ let
     @test isa(ptr, Ptr{Cvoid})
     @test arr[1] == '0'
 end
+
+# issue #34061
+sysimg = unsafe_string(Base.JLOptions().image_file)
+o_file = tempname()
+run(`$(Base.julia_cmd()) -J$sysimg --output-o=$o_file -e 'Base.reinit_stdio();
```

Comment:
```suggestion
run(`$(Base.julia_cmd()) --output-o=$o_file -e 'Base.reinit_stdio();
```

Reply:
Oh yeah, that's already included in `julia_cmd`